src/main/scala-2.12/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -104,11 +104,33 @@ object JmapMessages {
       subject = Subject(),
       textBody = TextBody())
 
-  def listMessages() =
+  def openpaasListMessageFilter(mailboxesKey: List[String]): Map[String, Any] = {
```

Comment:
Can you create a dedicated type instead of a `Map`?

Reply:
Is a type alias good enough for you ?

Reply:
yes